README.md

```diff
@@ -1,27 +1,17 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# My first review
 
+> this project is to learn how to review project.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
```

Comment:
- [ ] [OPTIONAL] I think it will be great if you suggest to the student that he removes the placeholder live-demo link or add the original live demo link